rest_framework-stubs/parsers.pyi

```diff
@@ -8,6 +8,8 @@ from rest_framework.renderers import JSONRenderer
 _Data = TypeVar("_Data")
 _Files = TypeVar("_Files")
 
+_Validator = Callable[[Any], None]
```

Comment:
unused?

Reply:
Ah yeah, leftover. Will open separate pull request for this.